packages/animations/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: animations
 description: Fancy pre-built animations that can easily be integrated into any Flutter application.
-version: 2.0.0-nullsafety.0
+version: 2.1.0-nullsafety.0
```

Comment:
Since 2.0.0 is still in pre-release, I don't think we need to bump the MINOR version for this. It can just be part of the final 2.0.0 release.

Reply:
Good point, I changed the version to `2.0.0-nullsafety.1`